test/unit/presenters/formats/edition_format_presenter_test.rb

```diff
@@ -62,18 +64,10 @@ def result
       assert_equal "override me", result[:document_type]
     end
 
-    context "when in beta" do
-      should "include phase" do
-        edition.expects(:in_beta).returns(true)
-        assert_equal "beta", result[:phase]
-      end
-    end
-
-    context "when not in beta" do
-      should "not include phase" do
-        edition.expects(:in_beta).returns(false)
-        assert_not_includes result.keys, :phase
-      end
+    should "include phase name" do
```

Comment:
Are we missing a test here now?

Reply:
you mean the "should not include phase" test?

Reply:
Yeah, what happened to that?

Reply:
I'll add it back in.

Reply:
done